src/blop/de/de_opt_utils.py

```diff
@@ -211,7 +211,7 @@ def generate_hardware_flyers(
             times_dict[motor_name] = time_
         times_list.append(times_dict)
 
-    for param, vel, time_ in zip(population, velocities_list, times_list):
+    for param, vel, time_ in zip(population, velocities_list, times_list, strict=False):
```

Comment:
What does `strict=False` do in this case?

Reply:
Apparently it stops zipping after the shortest list is exhausted according to: https://peps.python.org/pep-0618/

The Ruff linter made this change automatically.